src/libraries/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using Xunit;
 
 [assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
Is this one still a problem?

Reply:
I'll try that as well, just wanted to get a PR up first.